### PR TITLE
fix(content): Remnant: Fixing a confusing block message in Cognizance 32

### PR DESCRIPTION
**Pull request by Zitchas**
**Bugfix:** This PR addresses issue #{{insert number}}

## Fix Details
I'm removing the cargo space requirement and adding to the mission description that they need to pick up 58 tons of material. Not quite as obvious as reserving the space, but hopefully people actually read it.

## Background
Sort of a problematic situation: The reserved cargo was appropriate for (and expects) that the player is switching to the Peregrine that they are given during this mission, so the amount of cargo their current ship has is actually irrelevant and should not block the mission. But there's no way to tell the mission that it needs to reserve that cargo space at the *end* of the "on accept" instead of at the start of the "on offer."

Thanks to Rocky196884 on Discord, among others, for reporting this bug.

## Testing Done
n/a

## Save File
n/a

### CODE REVIEW

File: data/remnant/remnant 2 cognizance.txt [hunk 1]
```diff
@@ -1349,10 +1349,9 @@ mission "Remnant: Cognizance 31"
 
 mission "Remnant: Cognizance 32"
 	name "Pickup Refined Fuel"
```
[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant 2 cognizance.txt:1351]: Thanks! Will change shortly.

[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant 2 cognizance.txt]: Done, fixed. Thanks!

File: data/remnant/remnant 2 cognizance.txt [hunk 2]
```diff
@@ -1427,7 +1426,7 @@ mission "Remnant: Cognizance 33"
 	on offer
 		event "remnant: remove hidden"
 		conversation
-			`Your reception is as friendly as <origin>'s reputation suggests: No one is there to meet you, and no one arrives to help load the cargo. All that greets you is a forlorn datapad mounted on a post in front of the pile of crates, requesting that you enter your pickup code to disarm the security sensors, as well as a statement that ships depart as soon as they are done loading. You enter the pickup code Chilia had sent you while you were in flight, load the <ship>, and get ready to head to <planet>.`
+			`Your reception is as friendly as <origin>'s reputation suggests: No one is there to meet you, and no one arrives to help load the cargo. All that greets you is a forlorn datapad mounted on a post in front of the pile of crates, requesting that you enter your pick up code to disarm the security sensors, as well as a statement that ships depart as soon as they are done loading. You enter the code Chilia had sent you while you were in flight, load the <ship>, and get ready to head to <planet>.`
```
Comment (tehhowch) [inline data/remnant/remnant 2 cognizance.txt:1429]: The modified sentence is rather lengthy. What about

```suggestion
			`Your reception is as friendly as <origin>'s reputation suggests: No one is there to meet you, and no one arrives to help load the cargo. All that greets you is a forlorn security console near the pile of crates and a posted statement directing ships to depart as soon as they finish loading. You use the code Chilia had sent to unlock the crates, load the <ship>, and get ready to head to <planet>.`
```

**Review by tehhowch — COMMENTED** [review]
I'm sure there's lots of text in lots of missions that can similarly be trimmed, but I definitely don't have time to do copy edits 😉

**Pull request merged**